js/templates/netdev/netdevmacs.js

```diff
@@ -0,0 +1,251 @@
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2021 LMS Developers
+ *
+ *  Please, see the doc/AUTHORS for more information about authors!
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ *  $Id$
+ */
+
+var macAction = '?m=netdevmac';
+var macNetDevId = $('#mac-netdevid').val();
+var netdevmactable = $('#netdevmactable');
+var netdevmactableHTML = ' #netdevmactable';
+var netdevmactableHeader = $('#netdevmacs .lms-ui-tab-header');
+var netdevmactableHeaderHTML = ' #netdevmacs .lms-ui-tab-header';
+
+function getMacs() {
+    netdevmactable.load(netdevmactableHTML);
+    netdevmactableHeader.load(netdevmactableHeaderHTML);
+    hideAddManagementMac();
+}
+
+function showLabels(macid, label) {
+    $.ajax({
+        url: macAction + '&api=1&oper=showlabels',
+        async: true,
+        method: 'POST',
+        dataType: 'json'
+    }).done(function (data) {
+        var selectHTML = $('<select name="label_edit_' + macid + '"' +
+            'id="label_edit_' + macid + '"' +
+            'class="lms-ui-combobox" required></select>');
+        var macrow = $('#netdevmactable [data-macid="' + macid + '"] .label');
+
```

Comment:
Poprawka niekonieczna, ale warto zdawać sobie sprawę w przyszłości - lepiej od razu cały kod HTML przygotować i od razu go jednym wywołaniem wstawić niż robić to na raty poprzez tak naprawdę DOM - to jest po prostu wolniejsze.